plasmapy/physics/parameters.py

```diff
@@ -380,7 +380,7 @@ def thermal_speed(T, ion="e"):
     >>> from astropy import units as u
     >>> thermal_speed(5*u.eV, 'p')
```

Comment:
Do you think you could add an example for the default (electron) value here?

Reply:
I think that's is covered in line 386: `thermal_speed(5*u.eV)`

Reply:
... huh, it is. Sorry, I missed it somehow!